ui-ngx/src/app/shared/components/ota-package/ota-package-autocomplete.component.ts

```diff
@@ -158,7 +160,8 @@ export class OtaPackageAutocompleteComponent implements ControlValueAccessor, On
         }),
         map(value => value ? (typeof value === 'string' ? value : value.title) : ''),
         switchMap(name => this.fetchPackages(name)),
-        share()
+        share(),
+        takeUntil(this.destroy$)
```

Comment:
This operator does not make sense because the control is done through the template and not manually.